.github/workflows/auto-merge-pr.yaml

```
@@ -0,0 +1,46 @@
+name: "Auto Merge Authorized PRs"
+
+on:
+  workflow_run:
+    workflows: ["Check Authorized Users"]
+    types:
+      - completed
+
+jobs:
+  auto-merge:
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'pull_request' }}
```

check authorized users runs on `pull_request_target`
```suggestion
    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```